### PR TITLE
🧹 Hide progressbars when debug logging is enabled

### PR DESCRIPTION
**Pull request by czunker**
Signed-off-by: Christian Zunker <christian@mondoo.com>

### CODE REVIEW

**Review by imilchev — APPROVED** [review]
Nice! Works as expected!

File: logger/log.go [hunk 1]
```diff
@@ -64,8 +64,10 @@ func Set(level string) {
 		zerolog.SetGlobalLevel(zerolog.InfoLevel)
 	case "debug":
 		zerolog.SetGlobalLevel(zerolog.DebugLevel)
+		Debug = true
```
Comment (chris-rock) [inline logger/log.go]: Using this variable has disadvantages. Settings this to true also generates a log of log files that are currently only set via `DEBUG=1`. Can we instead change `isatty.IsTerminal(os.Stdout.Fd()) && !logger.Debug` to something that checks the log-level?

Comment (czunker) [reply logger/log.go]: I exposed the log level and check for it.

File: explorer/scan/local_scanner.go [hunk 2]
```diff
@@ -198,7 +198,7 @@ func (s *LocalScanner) distributeJob(job *Job, ctx context.Context, upstreamConf
 		orderedKeys = append(orderedKeys, assetList[i].PlatformIds[0])
 	}
 	var multiprogress progress.MultiProgress
-	if isatty.IsTerminal(os.Stdout.Fd()) {
+	if isatty.IsTerminal(os.Stdout.Fd()) && !strings.EqualFold(logger.GetLevel(), "debug") {
```
Comment (chris-rock) [inline explorer/scan/local_scanner.go]: just additional challenge: also do not show it in trace :-)

Comment (czunker) [reply explorer/scan/local_scanner.go]: Fixed.

**Review by chris-rock — APPROVED** [review]
Thank you @czunker  Very clean.

**Pull request merged**